src/Features.php

```diff
@@ -156,6 +161,13 @@ public static function is_active( string $name ): bool {
 	 * Add hooks related to Features activation
 	 */
 	public static function hooks() {
+		add_action(
+			'cmb2_options-page_process_fields_' . Settings::METABOX_ID,
+			__CLASS__ . '::on_pre_process',
```

Comment:
You can also use the array notation to pass the callable. It has better IDE support.
```php
[ self::class, 'on_pre_process' ]
```
On PHPStorm for me the current notation using `__CLASS__` is not picked up as a usage, which causes unused elements warnings and doesn't allow for navigating to the declaration.